### PR TITLE
Show artifacts on build page

### PR DESCRIPTION
**Pull request by nakhbari**
Issue: #709

No download button yet, as we haven't figured out the backend for that yet.

#### Screenshot
image

```
- [x] I have run `rspec` and corrected all errors
- [x] I have run `rubocop` and corrected all errors
- [x] I have run `npm run test` and corrected all errors
- [x] I have tested this change locally and tried to launch the server as well as access a project, and with that at least one build
- [x] If there is an existing issue, make sure to add `Fixes ...` as part of the PR body to reference the issue you're solving
```

### CODE REVIEW

File: web/app/common/test_helpers/mock_build_data.ts [hunk 1]
```diff
@@ -38,7 +43,8 @@ export const mockBuildResponse: BuildResponse = {
   branch: 'test-branch',
   clone_url: 'https://github.com/nakhbari/HelloWorld.git',
   ref: 'pull/1/head',
-  sha: '5903a0a7d2238846218c08ad9d5e278db7cf46c7'
+  sha: '5903a0a7d2238846218c08ad9d5e278db7cf46c7',
+  artifacts: [mockBuildArtifactResponse_log, {id: '54321', type: 'hack.exe'}]
```
Comment (KrauseFx) [inline web/app/common/test_helpers/mock_build_data.ts]: gotta love this

File: web/app/models/build.ts [hunk 2]
```diff
@@ -37,23 +44,25 @@ export class Build {
...
-    this.buildTools = buildSummary.build_tools;
-    this.cloneUrl = buildSummary.clone_url;
-    this.branch = buildSummary.branch;
-    this.ref = buildSummary.ref;
-    this.duration = buildSummary.duration;
-    this.sha = buildSummary.sha;
+  constructor(build: BuildResponse) {
+    this.number = build.number;
+    this.projectId = build.project_id;
+    this.description = build.description;
```
Comment (KrauseFx) [inline web/app/models/build.ts]: I see `description` and `platform` are syntax highlighted, does this conflict with any system methods in JS?

Comment (nakhbari) [reply web/app/models/build.ts]: Hmm i just checked the reserved word list, don't see those on there. not sure why they're highlighted https://web.archive.org/web/20140902235313/http://javascript.about.com/library/blreserved.htm

**Comment by nakhbari** [comment]
@KrauseFx Thanks for the quick review ❤️

File: web/app/models/build.ts (hunk 2 above)
Comment (KrauseFx) [reply web/app/models/build.ts]: 👍

**Pull request merged**